_pages/docs/quick-start-guide/index.md

```diff
@@ -28,16 +28,17 @@ This quick start guide will show you how to make a simple petstore which returns
 17. Set the title to `Return a pet`
 18. Under &quot;Query Match Rules&quot;, type in `pet` as the key and `sammy` as the value
 19. Click &quot;+&quot; next to the entry
-20. Set the response status code to `200`
-21. Set the response body to `"Sammy the cat"` (with quotes)
-22. Click "Save"
-23. In order for the Mockdefinition to match requests, it has to be uploaded to the Mockdefinition server. Click on &quot;Export Mock to Server&quot; in the left-hand sidebar.
-24. Move the Mockdefinition that you&#39;ve created to the right-hand side using the arrows
-25. Use [http://localhost:5000/api/v1/OrbitalAdmin](http://localhost:5000/api/v1/OrbitalAdmin) as the server URI
-26. Click &quot;Upload&quot;
-27. If uploading was successful, it will say that it was successful after a few seconds and move the exported Mockdefinition to the left-hand side. If it was not successful, ensure that your server is running. For more information, refer to the Installation instructions and ensure that you can access &quot;[http://localhost:5001](http://localhost:5001)&quot; in your browser.
-28. Navigate to [http://localhost:5000/pets](http://localhost:5000/pets) in your browser. It should display `No pets found`, as we did not specify a pet.
-29. Navigate to [http://localhost:5000/pets?pet=sammy](http://localhost:5000/pets?pet=sammy) in your browser. It should display `Sammy the cat`.
+20. In "Request Match Rules", remove the "Accept All" match rule in the "Url Match Rules" section
```

Comment:
Just a little confused with this. does it mean that there is already a match rule in the new scenario?